Bindings/bindings.json

```diff
@@ -474,6 +474,124 @@
             ]
         },
         {
+            "type": "apiHubTable",
+            "displayName": "SaaS Table",
```

Comment:
since this is repeated, I wonder if you can't use the "variable" syntax reference used elsewhere?